examples/dask/ingest_ssh.py

```diff
@@ -0,0 +1,41 @@
+import dask
+from dask import dataframe
+import os
+from aperturedb import EntityDataCSV
+from dask.distributed import Client, SSHCluster, performance_report
+
+if __name__ == "__main__":
+    batchsize = 4000
+    numthreads = 14
+
+    # cluster = SSHCluster(
```

Comment:
should this be an optional argument or have a comment?

Reply:
Oh this was an unfinished experiment. I'll remove this file for now.